requirements-noaws.txt

```diff
@@ -0,0 +1,7 @@
+numpy>=1.16.0
```

Comment:
This duplicates `requirements.txt`, to avoid this it would be better to have instead a `requirement-aws.txt` that just contains the aws dependencies.